roles/baseline/tasks/main.yml

```diff
@@ -33,7 +32,7 @@
     - V4_CFG_NFS_SVR_PATH is defined
     - V4_CFG_MANAGE_STORAGE is defined
     - V4_CFG_MANAGE_STORAGE
-    - PROVIDER != "aws"
+    - CREATE_NFS_STORAGE_CLASS is defined
```

Comment:
Why the new variables CREATE_EFS_STORAGE_CLASS and CREATE_NFS_STORAGE_CLASS? They are unneeded and don't following the naming standard.